docs/source-2.0/additional-specs/smoke-tests.rst

```diff
@@ -0,0 +1,194 @@
+.. _smoke-tests:
+
+===========
+Smoke Tests
+===========
+
+Smoke tests are small, simple tests intended to uncover large issues by ensuring
+basic functionality works as expected. In Smithy, these tests are used to make
+sure clients can successfully make a request to a live service, and that the
+service responds as expected.
+
+
+--------
+Overview
+--------
+
+This specification defines a single trait in the ``smithy.test`` namespace that is
+used to make basic assertions about how the service should respond to a given
+request from the client.
+
+:ref:`smithy.test#smokeTests <smokeTests-trait>`
+    Defines a set of test cases to send to a live service to ensure that a
+    client can successfully connect to a service and get the right kind of
+    response.
+
+This trait can be used by client and service developers to make sure their clients
+can successfully connect to the service, and the service responds as expected. The
+trait can be used by client code generators to generate test cases that are run on
+the generated client against a service to ensure that this basic functionality is
```

Comment:
nit:

```suggestion
trait can be used by client code generators to generate test cases that test
generated clients against services to ensure that critical functionality is
```

Reply:
I completely reworded this section, let me know what you think